priv/perf/apps/load_test/lib/scenario/account_transactions.ex

```diff
@@ -0,0 +1,271 @@
+
+    session =
+      add_metric(
+        session,
+        {:call, {LoadTest.Scenario.AccountTransactions, '/transaction.create'}},
+        Timing.timestamp() - start
+      )
+
+    typed_data_signed = sign_tx(inputs, sign_hash, typed_data, sender)
+
+    start = Timing.timestamp()
+    {:ok, response} = Api.Transaction.submit_typed(Connection.client(), typed_data_signed)
+
+    session =
+      add_metric(
+        session,
+        {:call, {LoadTest.Scenario.AccountTransactions, '/transaction.submit_typed'}},
+        Timing.timestamp() - start
+      )
+
+    %{
+      "txhash" => tx_id
+    } = Jason.decode!(response.body)["data"]
+
+    wait_until_tx_sync_to_watcher(session, tx_id)
+  end
+
+  defp create_transaction(session, sender) do
+    {:ok, response} =
+      Api.Transaction.create_transaction(
+        Connection.client(),
+        %Model.CreateTransactionsBodySchema{
+          owner: Encoding.to_hex(sender.addr),
+          fee: %Model.TransactionCreateFee{
+            currency: Encoding.to_hex(@eth)
+          },
+          payments: [
+            %Model.TransactionCreatePayments{
+              amount: @test_output_amount,
+              currency: Encoding.to_hex(@eth),
+              owner: Encoding.to_hex(session.assigned.faucet.addr)
+            }
+          ]
+        }
+      )
+
+    %{
+      "result" => "complete",
+      "transactions" => [
+        %{
+          "inputs" => inputs,
+          "sign_hash" => sign_hash,
+          "typed_data" => typed_data,
+          "txbytes" => txbytes
+        }
+      ]
+    } = Jason.decode!(response.body)["data"]
+
+    {:ok, [inputs, sign_hash, typed_data, txbytes]}
+  end
+
+  defp get_balance(sender) do
+    {:ok, response} =
+      Api.Account.account_get_balance(
+        Connection.client(),
+        %Model.AddressBodySchema{
+          address: Encoding.to_hex(sender.addr)
+        }
+      )
+
+    {:ok, response.body}
+  end
+
+  defp get_utxos(sender) do
+    {:ok, response} =
+      Api.Account.account_get_utxos(
+        Connection.client(),
+        %Model.AddressBodySchema1{
+          address: Encoding.to_hex(sender.addr)
+        }
+      )
+
+    {:ok, response.body}
+  end
+
+  defp get_transactions(_sender) do
+    # There is an issue openapi generated client does not work well with optional request body
+    # https://github.com/OpenAPITools/openapi-generator/issues/5234
+    # So we are not filtering by sender now
+
+    {:ok, response} =
+      Api.Account.account_get_transactions(
+        Connection.client(),
+        %Model.GetAllTransactionsBodySchema{}
+      )
+
+    {:ok, response.body}
+  end
+
+  defp sign_tx(inputs, sign_hash, typed_data, sender) do
+    signature =
+      sign_hash
+      |> Encoding.to_binary()
+      |> Encoding.signature_digest(Encoding.to_hex(sender.priv))
+      |> Encoding.to_hex()
+
+    signatures = Enum.map(inputs, fn _ -> signature end)
+    typed_data_signed = Map.put_new(typed_data, "signatures", signatures)
```

Comment:
You can replace those two lines with `Map.put_new(typed_data, "signatures", signatures)` and it will be returned from the function.